vulkan_generator/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+py_library(
+    name = "vulkan_generator",
+    srcs = glob(
+        ["*.py"],
+        exclude = ["main.py"],
+    ),
+    srcs_version = "PY3",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//vulkan_generator/vulkan_parser",
+    ],
+)
+
+py_binary(
+    name = "main",
+    srcs = ["main.py"],
+    args = [
+        "$(location @vulkan-headers//:vk.xml)",
```

Comment:
Have we added vk.xml to thirdparty? Is this OK from a license perspective?

Reply:
One more thing is the version of this repo. As far as I can see Vulkan 1.2 repo being used which will not work with the current parser, as the XML has changed. We need to update it to version is compatible with our parser.

I believe we have not intention to support more than one version of Vulkan XML. Therefore, if we are going to use a unified Vulkan repo for all our Vulkan related work. It has to be the one that parser works with.

Reply:
We can update our vulkan_headers dep and get the latests.